src/lib/asn1/asn1_time.cpp

```diff
@@ -19,9 +19,11 @@ namespace Botan {
 /*
 * Create an X509_Time
 */
-X509_Time::X509_Time(const std::string& time_str)
+X509_Time::X509_Time()
+   : year(0), month(0), day(0),
+     hour(0), minute(0), second(0),
```

Comment:
You mean just in case for other constructors, right? Because for this constructor this is exactly what the initalizer list does.